### DISCUSSION TITLE
Webpart stopped loading, something went wrong error.

### DISCUSSION THREAD

**Discussion by rohitchouhan** [microsoft-search/pnp-modern-search#3578] [Q&A]
I am using Modern PnP serach v4 since long but suddenly since yesterday webparts are not loading and showing error.

<img width="605" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/41120251/9790ade7-7262-4e12-8db2-56dfd58662c2">

I have cleared the browser cache but its still not working. 
Please suggest and  assist with this issue as several application impacted.

**Comment by wobba** [comment]
As there is no change on the web part, then it is not a web part specific issue.

**Reply by rohitchouhan** [reply]
I am not able to resolve this issue. This is across the tenant.
any ways to troubleshoot this.?

**Reply by wobba** [reply]
Sometimes deleting the app package from the the App Catalog and re-adding it helps. This is a generic SPFx issue which may happen when using CDN's. Most commonly it happens when you add a new version of an app.
